pkgs/development/tools/reindeer/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, libiconv
+, darwin
+, stdenv
+, nix-update-script
+}:
+
+rustPlatform.buildRustPackage rec {
```

Comment:
Unlike [buck2](https://github.com/NixOS/nixpkgs/pull/232471#issuecomment-15534815400), we can and should use stable Rust.